growth/too/templates/gcn.jinja2

```diff
@@ -0,0 +1,9 @@
+{% from 'macros.jinja2' import oxford_commas %}
+
+SUBJECT: GROWTH follow-up of {{plan.event.gcn_notices[-1].stream}} trigger {{plan.dateobs}} with {{plan.telescope}}
+
+{% filter rewrap %}
+{{oxford_commas(authors)}} report on behalf of the ZTF project and the GROWTH collaboration:
```

Comment:
Please wrap these lines to about 80 characters so that the template is more readable.